widgetssdk/src/main/java/com/glia/widgets/launcher/EngagementLauncher.kt

```diff
@@ -0,0 +1,57 @@
+package com.glia.widgets.launcher
+
+import android.app.Activity
+import com.glia.androidsdk.Engagement
+
+/**
+ * An interface for launching different types of engagements, such as chat,
+ * audio calls, video calls, and secure conversations
+ */
+interface EngagementLauncher {
```

Comment:
If we do that, integrators will be able to pass queue IDs to each call of these functions. The Engagement Launcher aims to avoid such an ambiguity.
It will be the only entry point for interaction with the Widgets, which means we will have a single source of truth, one list of queues to interact with for all types of engagements.
For more detailed information, you can refer to the project [plan document](https://glia.atlassian.net/wiki/spaces/ENG/pages/3954769921/Glia+Entry+Widget+for+Mobile#Technical-Plan), Technical plan → first section.
Or we can make a quick sync.